mpc/commands/null_server

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+"""
+Script to launch a null server.  Press CTRL-C to stop
+"""
+
+from click import command, option
+
+
+@command()
+@option("--path", default=".", help="Server working path")
+def null_server(path: str) -> None:
```

Comment:
It is for checking basic server setup without generating data.  It would require docs and some kind of client to use in a production setting, so easier to just remove it for now.